etc/inc/allow-common-devel.inc

```diff
@@ -38,3 +38,10 @@ noblacklist ${HOME}/.bundle
 # Rust
 noblacklist ${HOME}/.cargo
 noblacklist ${HOME}/.rustup
+
+#Arduino
+noblacklist ${HOME}/.arduino15
+noblacklist ${HOME}/Arduino
+
+# PlatformIO (can be used with Arduino)
```

Comment:
Code looks fine. We do `sort` these groups `alphabetically`. Please move it accordingly.